src/components/js-tracer/Chart.js

```diff
@@ -8,20 +8,20 @@ import {
   TIMELINE_MARGIN_LEFT,
   TIMELINE_MARGIN_RIGHT,
   JS_TRACER_MAXIMUM_CHART_ZOOM,
-} from '../../app-logic/constants';
-import explicitConnect from '../../utils/connect';
-import { JsTracerCanvas } from './Canvas';
```

Comment:
For example we have this line that we should leave without changing because it starts with './'.

Reply:
Ok, I'll get on that!